system-tests/src/tests/organization-list.spec.ts

```diff
@@ -1,11 +1,80 @@
 
+import { manageOrganization } from "@/utils/organizationUtils"
+
 test.use({
   storageState: "src/states/admin@example.com.json",
 })
 
+test("create new organization, edit it and it's permissions, and delete it", async ({ page }) => {
+  await page.goto("http://project-331.local/")
+  await page.getByRole("link", { name: "All organizations" }).click()
+
+  // Create organization
+  await page.getByRole("button", { name: "Create a new organization" }).click()
+  await page.getByRole("textbox", { name: "Organization name" }).click()
+  await page.getByRole("textbox", { name: "Organization name" }).fill("createnewtestorganization")
+  await page.getByRole("textbox", { name: "Slug" }).click()
+  await page.getByRole("textbox", { name: "Slug" }).fill("createnewtestorganizationslug")
+  await page.getByTestId("dialog").getByRole("button", { name: "Create" }).click()
+  await page.getByRole("heading", { name: "createnewtestorganization" }).click()
+  await manageOrganization(page, "createnewtestorganization")
+  await page.getByText("createnewtestorganization", { exact: true }).click()
+  await page.getByText("createnewtestorganizationslug").click()
+
+  // Edit organization
+  await page.getByRole("button", { name: "Edit" }).click()
+  await page.getByRole("textbox", { name: "Name" }).fill("createnewtestorganizationedited")
+  await page.getByRole("textbox", { name: "Slug" }).fill("createnewtestorganizationslugedited")
+  await page.getByRole("button", { name: "Save" }).click()
+  await page.goto("http://project-331.local/")
+  await page.getByRole("link", { name: "All organizations" }).click()
+  await page.getByRole("heading", { name: "createnewtestorganizationedited" }).click()
+  await manageOrganization(page, "createnewtestorganizationedited")
+  await page.getByText("createnewtestorganizationedited", { exact: true }).click()
+  await page.getByText("createnewtestorganizationslugedited").click()
+
+  // Add user with a role
+  await page.getByRole("tab", { name: "Permissions" }).click()
+  await page.getByRole("button", { name: "Add user" }).click()
+  await page.getByRole("textbox", { name: "Email" }).click()
+  await page.getByRole("textbox", { name: "Email" }).fill("teacher@example.com")
+  await page.getByLabel("Role").selectOption("Teacher")
+  await page.getByRole("button", { name: "Save" }).click()
+  await page.getByText("Teacher", { exact: true }).click()
+  await page.getByText("teacher@example.com").click()
+  await page.getByText("Teacher Example").click()
+
+  // Edit teacher role
+  await page.getByRole("button", { name: "Edit user Teacher Example" }).click()
+  await page.getByTestId("dialog").getByLabel("Role").selectOption("CourseOrExamCreator")
+  await page.getByRole("button", { name: "Save" }).click()
+  await page.getByText("CourseOrExamCreator").click()
+  await page.getByText("teacher@example.com").click()
+  await page.getByText("Teacher Example").click()
+
+  // Delete teacher
+  page.once("dialog", (dialog) => {
```

Comment:
Once you switch the implementation to use the new confirm dialogs, you can then use these functions: https://github.com/rage/secret-project-331/blob/c0cca55bd346df216ba0f6cd4b0ab8096208cae3/system-tests/src/utils/dialogs.ts#L1-L114